portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +26,29 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+    private List<String> fact;
+    
+    @Override
```

Comment:
check if this can be a const.

A const is initialized at compile time. Init() are functions that should be used as a the last resort. This leads to dependencies in control flow, which is easier if avoided.

Reply:
changed public void init() => public DataServlet()